resources/EXM_124/Makefile

```diff
@@ -0,0 +1,45 @@
+all: setup-cqf-ruler setup-synthea generate-patients calculate-patients
+
+setup-cqf-ruler: build-cqf-ruler clone-connectathon seed-cqf-ruler
+
+build-cqf-ruler:
+	docker run --name cqf-ruler --rm -dit -p 8080:8080 contentgroup/cqf-ruler
+
+clone-connectathon:
+	git clone https://github.com/DBCG/connectathon.git
+
+seed-cqf-ruler:
+	until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-dstu3`; do printf '.'; sleep 5; done
+	curl -X POST \
+  http://localhost:8080/cqf-ruler-dstu3/fhir/Measure \
+  -H 'Content-Type: application/json' \
+	-d @connectathon/resources/fhir3/measure/measure-exm124-FHIR.json
+	curl -X POST \
+  http://localhost:8080/cqf-ruler-dstu3/fhir \
+  -H 'Content-Type: application/json' \
+	-d @connectathon/resources/fhir3/library/all-libraries-bundle.json
+	curl -X POST \
+  http://localhost:8080/cqf-ruler-dstu3/fhir \
+  -H 'Content-Type: application/json' \
+	-d @connectathon/resources/fhir3/valuesets/valuesets-bundle-exm124-FHIR.json
+
+setup-synthea: clone-synthea patch-synthea
+
+clone-synthea:
+	git clone https://github.com/synthetichealth/synthea.git
+
+patch-synthea:
+	cp patch/synthea.properties ./synthea/src/main/resources/
+	cp patch/breast_cancer_exm124.json ./synthea/src/main/resources/modules/
+
+PATIENT_COUNT := 10
+generate-patients:
+	cd synthea && ./run_synthea -a 51-74 -g F -p $(PATIENT_COUNT) -m breast*
```

Comment:
Minor, but let's specify the `_exm124.json` module in this call, since `breast*` will also pick up the original breast cancer module.

Reply:
Good call--that will probably reduce the amount of thrown away patients generated as well 👍